lib/src/https.rs

```diff
@@ -1443,6 +1443,7 @@ fn rustls_version_str(version: ProtocolVersion) -> &'static str {
         ProtocolVersion::DTLSv1_2 => "tls.version.DTLSv1_2",
         ProtocolVersion::DTLSv1_3 => "tls.version.DTLSv1_3",
         ProtocolVersion::Unknown(_) => "tls.version.Unknown",
+        _ => "unimplemented variant, please check Rustls to update the code",
```

Comment:
We have to implements those missing protocol version ;)

Reply:
The enum `ProtocolVersion` is [marked as non-exhaustive](https://docs.rs/rustls/latest/rustls/enum.ProtocolVersion.html):

> Non-exhaustive enums could have additional variants added in future. Therefore, when matching against variants of non-exhaustive enums, an extra wildcard arm must be added to account for any future variants.

There's nothing to do for now, we'll have to keep up with the updates.